daprdocs/content/en/contributing/contributing-overview.md

```diff
@@ -54,8 +54,48 @@ All contributions come through pull requests. To submit a proposed change, follo
     - Code changes require tests
 1. Update relevant documentation for the change
 1. Commit and open a PR
-1. Wait for the CI process to finish and make sure all checks are green
-1. A maintainer of the project will be assigned, and you can expect a review within a few days
+
+    - Developer Certificate of Origin: Signing your work
```

Comment:
I think embedding such a big chunk of information within a bullet in the PR section might be a bit overwhelming. Suggest making DCO its own top level section at the same level as and immediately preceding `## Pull Requests` - see what I did in [dapr/dapr PR#4018](https://github.com/dapr/dapr/pull/4018/files) for an example.